### PR TITLE
DG - Done function and Storage Component

### LINKED ISSUE #113: Complete DG section 3.6
Complete the DG section pertaining to the Storage class

### PR DESCRIPTION
**Pull request by yAOwzers**
UHm Sequence diagram for mark Done function is still underway... there were some technical difficulties along the way.......

### CODE REVIEW

**Review by franceneee — APPROVED** [review]
looks okay!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -294,13 +294,15 @@ The `Page` object,
 
 ### 3.6. Storage Component
 
-/* to insert UML diagram */
+![UML diagram for Storage](/diagrams/class/jpeg/Storage_UML_class.jpg)
```
Comment (Lusi711) [inline docs/DeveloperGuide.md]: Should be diagrams/class/jpeg/Storage_UML_class.jpg

Comment (franceneee) [reply docs/DeveloperGuide.md]: It still works actually xD

Comment (yAOwzers) [reply docs/DeveloperGuide.md]: Oh shoot okay, LEMME edit

**Pull request merged**